jenkinsapi/jenkins.py

```diff
@@ -327,16 +327,23 @@ def get_plugins_url(self, depth):
         # This only ever needs to work on the base object
         return '%s/pluginManager/api/python?depth=%i' % (self.baseurl, depth)
 
-    def install_plugin(self, plugin):
+    def install_plugin(self, plugin, restart=False, wait_for_reboot=False):
```

Comment:
Why `wait_for_reboot` is `False` by default? Perhaps it would be safer to wait.

Reply:
I just reused the existing flags from `install_plugins`. Will change to `True`.

Reply:
The `safe_restart` method also has `wait_for_reboot` on `False` by default. I don't really know why, so I'll change it to `True`.